<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>ADC</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="Setup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>241</width>
      <height>251</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QCheckBox" name="Ch0">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ch0</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Ch7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ch7</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Ch4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ch4</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Ch3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ch3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Ch5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ch5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sample Rate (us)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ch6_samrate">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="Ch1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ch1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Ch45">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="ch4_samrate">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ch2_samrate">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="Ch67">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="ch1_samrate">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="Ch6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ch6</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Ch2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ch2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ch3_samrate">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="Ch01">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="ch7_samrate">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="Ch23">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="ch5_samrate">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ch0_samrate">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="ch0_radio">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="ch1_radio">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="ch2_radio">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="ch3_radio">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="ch4_radio">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="ch5_radio">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="ch6_radio">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="ch7_radio">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="up">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>51</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="down">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>102</y>
        <width>51</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QPushButton" name="Start">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Stop">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOpen_Preferences"/>
    <addaction name="actionSave_Preferences"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_Preferences">
   <property name="text">
    <string>Save Preferences</string>
   </property>
  </action>
  <action name="actionOpen_Preferences">
   <property name="text">
    <string>Open Preferences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
